hack/simple-demo.sh

```diff
@@ -25,7 +25,16 @@ EAg+paWtlme2KNcostCsmIbwz+bc2aFV+AxCOpRjRpp3vYrbS5KhkmgC1Q==
 POLICY='{
   "publicKey": "'${PUBLIC_KEY//$'\n'/\\n}'",
   "sources": [
-    "github.com/hacbs-contract/ec-policies//policy"
+    {
+      "name": "EC Policies",
+      "policy": [
+        "github.com/hacbs-contract/ec-policies//policy/lib",
```

Comment:
Oh interesting! Does this actually work??

Reply:
Yeah, I wanted to see if it worked like this, and seems like it does.